lib/mdl.rb

```diff
@@ -112,6 +114,10 @@ def self.run(argv = ARGV)
               'description' => rule.description,
               'docs' => rule.docs_url,
             }
+          elsif Config[:checkstyle]
+            results_by_filename << <<~HEREDOC
+              <error line="#{line}" severity="error" message="#{rule.description}" source="#{rule.docs_url}" />
```

Comment:
this is error prone. How do you know there's not quotes in the line? This would break the XML parsing.